libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/route/NavigationMapRoute.java

```diff
@@ -139,20 +188,23 @@ public NavigationMapRoute(@Nullable MapboxNavigation navigation, @NonNull MapVie
   /**
    * Construct an instance of {@link NavigationMapRoute}.
    *
-   * @param navigation an instance of the {@link MapboxNavigation} object. Passing in null means your route won't
-   *                   consider rerouting during a navigation session.
+   * @param navigation an instance of the {@link MapboxNavigation} object. Passing in null means
+   *                   your route won't consider rerouting during a navigation session.
    * @param mapView    the MapView to apply the route to
    * @param mapboxMap  the MapboxMap to apply route with
    * @param styleRes   a style resource with custom route colors, scale, etc.
    * @param belowLayer optionally pass in a layer id to place the route line below
    */
   public NavigationMapRoute(@Nullable MapboxNavigation navigation, @NonNull MapView mapView,
-                            @NonNull MapboxMap mapboxMap, @StyleRes int styleRes, @Nullable String belowLayer) {
+                            @NonNull MapboxMap mapboxMap, @StyleRes int styleRes,
+                            @Nullable String belowLayer) {
     this.styleRes = styleRes;
     this.mapView = mapView;
     this.mapboxMap = mapboxMap;
     this.navigation = navigation;
     this.belowLayer = belowLayer;
+    featureCollections = new ArrayList<>();
+    alternativesVisible = true;
```

Comment:
Should we default this to true?  Maybe in `addRoutes()` if there are more than one route, we can default this to true?

Reply:
Essentially that's what it does if only one route is provided in the list, the alternative routes obviously won't be visible. If the user never wants to display alternatives they simply don't request alternatives in the nav request. Does that make sense?

Reply:
@cammace Okay yeah perfect, thanks!